src/Contexts/UserContext.jsx

```diff
@@ -22,11 +23,12 @@ const UserProvider = ({ children }) => {
           const {
             data: { watchlater },
           } = await axios.post("/api/user/watchlater", { video }, { headers: { authorization: authState.token } });
-          console.log(watchlater);
         } catch (err) {
-          console.log(err);
+          toast.error("Failed to Add");
```

Comment:
This is good. I only console log the errors but doing this is a great practice. Well done.